### PR TITLE
use waitress, not gunicorn

### PR DESCRIPTION
**Pull request by catscanner**

### CODE REVIEW

File: flyby/cli.py [hunk 1]
```diff
@@ -6,16 +6,13 @@
 from flyby.service import Service
 from flyby.server import app
 from sys import exit
-try:
-    from flyby.gunicorn import StandaloneApplication
-except ImportError:
-    logger.info("Unable to import StandaloneApplication from gunicorn, Windows is not supported")
-    exit(0)
+from datetime import datetime
```
Comment (gjtempleton) [inline flyby/cli.py]: This isn't needed/used as far as I can see.

**Comment by TransactCharlie** [comment]
interesting -- not used waitress before

File: flyby/cli.py (hunk 1 above)
Comment (catscanner) [inline flyby/cli.py]: Seems you're correct

File: requirements.txt [hunk 2]
```diff
@@ -7,3 +7,4 @@ validator.py==1.2.5
 pyyaml==3.12
 ntplib==0.3.3
 gunicorn==19.6.0
```
Comment (balintzs) [inline requirements.txt]: I guess this isn't needed anymore either

Comment (catscanner) [inline requirements.txt]: oops

**Comment by gjtempleton** [comment]
LGTM

**Pull request merged**